### ISSUE TITLE
Cannot find module '@mui/linaria-processor/sx'

### ISSUE THREAD

**Issue by denisso** [brijeshb42/next-app-linaria#1]
version 0.1.0

git clone
npm i
npm run dev

Error Cannot find module '@mui/linaria-processor/sx'

package.json
```
"@mui/linaria-processor": "file:../mui/material-ui/packages/mui-linaria-processor/build",
```

next.config.js
```
    tagResolver(_importPath, importName) {
      switch (importName) {
        case "styled":
          return "@mui/linaria-processor/styled";
        case "sx":
          return "@mui/linaria-processor/sx";
        default:
          return null;
      }
    },
```
